examples/bri-3/.env.sample

```diff
@@ -5,12 +5,12 @@
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
 DATABASE_URL="postgresql://postgres:example@localhost:5432/postgres"
-GOERLI_RPC_URL=""
-GOERLI_SERVICE_DID="did:ethr:0x5:0x0aE6F58120E3e53E94FeB7BFeAA11C7CFF6F70F0"
-GEORLI_SERVICE_SIGNER_PRIVATE_KEY=""
-SERVICE_URL="bri-3"
+GOERLI_RPC_URL="" # Any RPC url i.e. "https://rpc.ankr.com/eth_goerli". This is needed in order to 
+GOERLI_SERVICE_DID="did:ethr:0x5:<bpi_subject_public_key>" # bpi_subject_public_key = public key of the bpi subject that is performing the login 
+GEORLI_SERVICE_SIGNER_PRIVATE_KEY="<bpi_subject_private_key>" # bpi_subject_private_key = private key of the bpi subject that is performing the login
```

Comment:
It is confusing because it contains GOERLI in the name and the private key is agnostic to the blockchain network. How would this work in production? We cant expect from anyone to provide us with a private key for login purposes.